src/js/providers/html5.js

```diff
@@ -143,10 +143,10 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
         },
 
         seeking() {
-            const offset = _seekOffset !== null ? _seekOffset : _this.getCurrentTime();
+            const offset = _seekToTime !== null ? _convertTime(_seekToTime) : _this.getCurrentTime();
```

Comment:
Is it OK that `_convertTime(_seekToTime)` can also result in a seek?

Reply:
`_convertTime` never results in a seek. We don't know when Safari updates it's `seekable` TimeRange, so we use this method to update the DVR state vars in `updateDvrPosition` if the range has changed.

`_convertTime` is called every time we need a time `event.position` value which is calculated using these DVR state vars so it's perfect for keeping up with changing segment window.

Reply:
_converTime calls updateDvrPosition, which can call`                     _videotag.currentTime = _seekToTime; `

Reply:
It does not. That is code from `pause` (click the expand button).